scripts/setup-and-run-repoman.sh

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env bash
+# Maintainer: Andrew Ammerlaan <andrewammerlaan@riseup.net>
+#
+# This sets up repoman and runs the latest version
+
+# Enable colour output
+export TERM=xterm-color
+
+### Setup prerequisites
+python3 -m pip install --upgrade pip
+pip install lxml pyyaml
+sudo groupadd -g 250 portage
+sudo useradd -g portage -d /var/tmp/portage -s /bin/false -u 250 portage
+
+### Sync the portage repository
+git clone https://github.com/gentoo/portage.git
```

Comment:
what's the base image ? We sure `git` is installed ?

Reply:
Presumably git is installed since we are using `actions/checkout@v2` as well. In any case that part seems to be working. There are two things which I'm still working on:
- colour output
- figure out why `--jobs=2` makes it slower instead of faster